pkg/api/server.go

```diff
@@ -100,7 +100,8 @@ func (s *Server) registerHandlers() {
 	s.router.HandleFunc("/", s.indexHandler).HeadersRegexp("User-Agent", "^Mozilla.*").Methods("GET")
 	s.router.HandleFunc("/", s.infoHandler).Methods("GET")
 	s.router.HandleFunc("/version", s.versionHandler).Methods("GET")
-	s.router.HandleFunc("/echo", s.echoHandler).Methods("POST")
+	s.router.HandleFunc("/echo", s.echoHandler).Methods("POST,PUT")
+	s.router.HandleFunc("/echo/{echo:.*}", s.echoHandler).Methods("POST,PUT")
```

Comment:
Suggestions:
* What about using [PathPrefix()](https://pkg.go.dev/github.com/gorilla/mux#Route.PathPrefix) to allow "/echo/{anyPath}"?
* What about accepting all HTTP methods, e.g. not restricting these paths to specific methods?

```suggestion
	s.router.Path("/echo").HandlerFunc(s.echoHandler)
	s.router.PathPrefix("/echo/").HandlerFunc(s.echoHandler)
```